docs/labwc-config.5.scd

```diff
@@ -448,6 +448,7 @@ extending outward from the snapped edge.
 	- ActiveWindow - titlebar of active window
 	- InactiveWindow - titlebar of all windows that aren't focused by the
 	  cursor
+	- MenuHeader - menu header font ie separator labels
```

Comment:
Suggest simplifying to `- MenuHeader - menu title` to make it consistent with the other descriptions.